install-php-extensions

```diff
@@ -973,6 +980,63 @@ compareVersions() {
 	fi
 }
 
+# Install Oracle Instant Client & SDK
+installOracleInstantClient() {
+	if test $(php -r 'echo PHP_INT_SIZE;') -eq 4; then
+		installOracleInstantClient_client=client
+		if test "$DISTRO" = 'alpine'; then
+			# I couldn't find a way to install libresolv.so on Linux - let's use an old version that doesn't require it
+			installOracleInstantClient_version='11.2'
+			installOracleInstantClient_ic=https://raw.githubusercontent.com/pwnlabs/oracle-instantclient/master/instantclient-basic-linux-$installOracleInstantClient_version.0.4.0.zip
+			installOracleInstantClient_sdk=https://raw.githubusercontent.com/pwnlabs/oracle-instantclient/master/instantclient-sdk-linux-$installOracleInstantClient_version.0.4.0.zip
+		else
+			installOracleInstantClient_version='19.9'
+			installOracleInstantClient_ic=https://download.oracle.com/otn_software/linux/instantclient/199000/instantclient-basic-linux-$installOracleInstantClient_version.0.0.0dbru.zip
+			installOracleInstantClient_sdk=https://download.oracle.com/otn_software/linux/instantclient/199000/instantclient-sdk-linux-$installOracleInstantClient_version.0.0.0dbru.zip
+		fi
+	else
+		installOracleInstantClient_client=client64
+		if test "$DISTRO" = 'alpine'; then
+			# I couldn't find a way to install libresolv.so on Linux - let's use an old version that doesn't require it
```

Comment:
@mlocati https://github.com/atk4/image/blob/89433b7b29010b9f7046498ba745904af439e476/Dockerfile.template#L54 is this a solution for it?

Reply:
IDK, I'll give it a try

Reply:
It seems to work: see #220 (and thanks!)